hana/lib/HANAService.js

```diff
@@ -426,7 +426,7 @@ class HANAService extends SQLService {
           q.as = q.SELECT.from.as
         }
 
-        if (rowNumberRequired || q.SELECT.columns.length !== aliasedOutputColumns.length) {
+        if (cds.env.features.sql_simple_queries || rowNumberRequired || q.SELECT.columns.length !== aliasedOutputColumns.length) {
```

Comment:
wouldn't it be enough to only do this for `orderBy` or `having` clauses? Or are there more cases where the nesting is really needed?